README.md

```diff
@@ -32,3 +29,8 @@ Installation using CONDA
 Note that some examples requires optional dependencies for plotting, they are:
 - wxPython
 - matplotlib
+
+FMU Import Compliance
+-----------
+PyFMI is tested on an everyday basis both with the unit tests found in the `tests` directory but also extensively outside of this project with the `Optimica Compiler Toolkit` (OCT) provided by [Modelon](https://help.modelon.com/latest/reference/oct/). All models from the libraries provided by Modelon have been tested via simulation tests (which covers over 30000+ Modelica models) as FMUs (using the FMU Export functionality from OCT). PyFMI is the default execution engine for [Modelon Impact](https://modelon.com/modelon-impact/).
```

Comment:
PyFMI contains a number of FMUs used in these unit tests, we should some references to the generating tool for these.

Reply:
Made some updates, see my latest commit.

Reply:
This is valid for the examples/files/FMUs, but not for the ones in test/*, these do not have any ReadMe. Aside from that, I think there would still be value in having this information directly accessible here.

Also, thoughts on maybe formatting this via some brief bullets? I.e., making the text more factual and less conversational?